CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Kubeflow Examples
+
+You want to contribute to Kubeflow examples? That's awesome! Please refer to the short guide below. 
+
+# Contributing Guide
+
+The [Kubeflow](https://github.com/kubeflow/kubeflow/blob/master/README.md) project is dedicated to making machine learning on Kubernetes simple, portable and scalable. We need your support in making
+this repo the destination for top models and examples, which show the power of Kubeflow. We have created an initial list of
+proposed manifests. Please feel free to self-assign these examples, by following a simple 3 step process:
```

Comment:
Why call them manifests and not examples?

Reply:
Okay fixed. Don't want to sound stuck up but want to call out that these are canonical examples / artifacts and not mere some random examples. Models sounds a better word though even that doesn't capture the workflow part of these efforts. Ideas?